docs/sources/administration/image_rendering.md

```diff
@@ -39,7 +42,13 @@ You can install it using grafana-cli:
 grafana-cli plugins install grafana-image-renderer
 ```
 
-For further information and instructions refer to the [plugin details](https://grafana.com/grafana/plugins/grafana-image-renderer).
+For further information and instructions refer to [troubleshooting](#troubleshooting) and the [plugin details](https://grafana.com/grafana/plugins/grafana-image-renderer).
+
+#### Install in Grafana docker image
+
+This plugin does not currently work if it is installed in Grafana docker image. Since you are already running Grafana in a container it's suggested you run the image renderer in a container as well, see [Remote rendering service](#remote-rendering-service) for reference.
```

Comment:
"in the Grafana Docker image"

Reply:
I take this as you mean `s/it is installed in Grafana docker image/it is installed in the Grafana docker image`?

Reply:
Yeah, for consistency, since it's worded that way otherwise.